Kernel/Output/HTML/Layout.pm

```diff
@@ -1741,7 +1743,7 @@ sub Footer {
     my ( $Self, %Param ) = @_;
 
     my $Type          = $Param{Type}           || '';
-    my $HasDatepicker = $Self->{HasDatepicker} || 0;
+    my $picker = $Self->{HasDatepicker} || 0;
```

Comment:
This is a lexical variable. The name `$picker` is perfectly fine in Perl. But the OTOBO coding conventions state that variables should be in camel case.